tests/drivers/i2s/i2s_speed/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
```

Comment:
can you please the indentation here, 2 space for yaml

Reply:
Looking at other `testcase.yaml` files, e.g. from i2c test I understand that the problem was with
```
  - test:
```
line. This is fixed now.